messaging/Controllers/BundlesController.cs

```diff
@@ -118,34 +134,43 @@ public async Task<ActionResult<Bundle>> GetOutgoingMessageItems(string jurisdict
                     // Only show the next link if there are additional messages beyond the current message set

```

Comment:
During testing I tried pagination when certificateNumber and deathYear are provided as params and _count=1. The links returned did not specify page numbers etc. for pagination.
The if statement above also needs to check if certificateNumber == null and deathYear == null